src/WebJobs.Script/Diagnostics/MetricEventNames.cs

```diff
@@ -36,7 +36,7 @@ public static class MetricEventNames
         public const string InitializeHostConfiguration = "hostjsonfileconfigurationsource.loadhostconfigurationsource.initializehostconfig";
 
         // LanguageWorkerChannel events
-        public const string FunctionLoadRequestResponse = "languageworkerchannel.functionloadrequestresponse";
+        public const string FunctionLoadRequestResponse = "rpcworkerchannel.functionloadrequestresponse";
```

Comment:
I think we need to be careful on changing these event names - can we let PM's/Jason/Cindy and others who are writing detectors know that this is changing? (might be safer to leave it unchanged too)

Reply:
thanks! these are new metrics that @yojagad added recently. No dependencies yet. So changing them before analytics starts using them.